### ISSUE TITLE
Wraparound free convolution

### ISSUE THREAD

**Issue by roflmaostc** [bionanoimaging/FourierTools.jl#24]
Just as a note and how we could implement it:

```julia
function conv(u, v, wrap_free=true)
      pad u to twice the size
      v needs to be known for the full size
      convolve it 
      take center part of original size of u
```

Important is, that `v` needs to be known in the double field size if `v` is not 0 anyway on the border.

**Comment by RainerHeintzmann** [comment]
Yes, this would be the classical way. Yet, I wonder 1.) if we can avoid the padding. If you think about an FFT, can we not avoid the first Butterfly step or so by knowing which part is zero? 2.) if it is smart to use CZT since this can zoom. Maybe running it backwards automatically zero-pads?

**Comment by roflmaostc** [comment]
By looking at this, you don't much I  guess
https://www.researchgate.net/publication/4204073_On_Cooley-Tukey_FFT_method_for_zero_padded_signals/link/57455fcd08ae9f741b408c3e/download

**Comment by RainerHeintzmann** [comment]
... not worth the trouble, I guess.
